pkg/services/org/orgimpl/store.go

```diff
@@ -490,6 +491,25 @@ func (ss *sqlStore) UpdateOrgUser(ctx context.Context, cmd *org.UpdateOrgUserCom
 			return org.ErrOrgUserNotFound
 		}
 
+		// check if the user is external synced
+		// if so, we do not want to change the basic role
+		if orgUser.Role != cmd.Role {
+			authtablesess := sess.Table("user_auth")
+			var userAuth login.UserAuth
+			_, err := authtablesess.Where("id=?", cmd.UserID).Get((&userAuth))
+			if err != nil {
+				return err
+			}
+			fmt.Printf("userAuth: %v", userAuth)
+			fmt.Printf("cmd: %v", cmd)
```

Comment:
careful not to merge debug prints

Reply:
Yea I will not Merge until I get the backend ready. It's work in progress. 

Il mark it as Draft